deploy/terraform-datacommons-website/modules/gke/configure_cluster.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+gcloud container clusters get-credentials $CLUSTER_NAME --region $REGION
```

Comment:
Will this file be changed to terraform resources?

Reply:
Ideally yes, piece by piece. Difference between this and terraform resources is that by converting to Terraform resources, Terraform knows what it means to upgrade and destroy the infra. For the initial use case of automating the setup though, this may be less important.

Reply:
that's good. For one time setup, a script is sufficient.